src/components/InstallationPage.jsx

```diff
@@ -1,5 +1,70 @@
 import React from 'react'
+import { detect as detectBrowser } from 'detect-browser'
+import Wrapper from 'components/Wrapper'
+import NarrowContent from 'cozy-ui/transpiled/react/NarrowContent'
+import browserExtensionIcon from 'assets/browser-extension.svg'
+import Stack from 'cozy-ui/transpiled/react/Stack'
+import { MainTitle, Text } from 'cozy-ui/transpiled/react/Text'
+import Card from 'cozy-ui/transpiled/react/Card'
+import { ButtonLink } from 'cozy-ui/transpiled/react/Button'
+import { OrderedList, ListItem } from 'cozy-ui/transpiled/react/OrderedList'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import compose from 'lodash/flowRight'
+import { withClient } from 'cozy-client'
+import snarkdown from 'snarkdown'
 
-const InstallationPage = () => <div>Installation Page</div>
+const browser = detectBrowser()
+
+const stores = {
+  chrome:
+    'https://chrome.google.com/webstore/detail/cozy-personal-cloud/jplochopoaajoochpoccajmgelpfbbic',
+  firefox: 'https://addons.mozilla.org/en-US/firefox/addon/cozy-personal-cloud'
+}
+
+const DumbInstallationPage = props => {
+  const { t, client } = props
+
+  const cozyURL = new URL(client.getStackClient().uri)
```

Comment:
Nit: We should a helper for this directly on the client. client.getURL() ?

Reply:
That would save a few keystrokes and make code easier to read, yes :+1: